tabular/src/autogluon/tabular/predictor/interpretable_predictor.py

```diff
@@ -0,0 +1,98 @@
+import logging
+
+import numpy as np
+import pandas as pd
+
+from .predictor import TabularPredictor
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: Forcibly disable bagging, stacking, calibration, and non-interpretable models
+class InterpretableTabularPredictor(TabularPredictor):
+    def fit(self,
+            train_data,
+            tuning_data=None,
+            time_limit=None,
+            *,
+            presets='interpretable',
+            fit_weighted_ensemble=False,
+            calibrate=False,
+            **kwargs):
```

Comment:
nit: data types; add docstrings

Reply:
keeping to align with TabularPredictor, we can add type hints to TabularPredictor later on, but the type hints are already in TabularPredictor's docstring, which this method inherits.